tests/testthat/test-make-traceability-matrix.R

```diff
@@ -122,6 +123,30 @@ describe("Traceability Matrix", {
       trac_mat %>% tidyr::unnest(test_files) %>% pull(test_files) %>% unique(),
       "test-myscript.R"
     )
+
+    # Test faulty R/ directory
+    new_pkg_source_dir <- tempfile(pattern = "get_toplevel_assignments-")
+    fs::dir_create(new_pkg_source_dir)
+    on.exit(fs::dir_delete(new_pkg_source_dir), add = TRUE)
```

Comment:
Okay as is, but using `withr::local_tempdir` would automatically take care of lines 129 and 130 for you.